string2.py

```diff
@@ -37,8 +41,12 @@ def verbing(s):
 
 
 def not_bad(s):
-    # your code here
-    return
+    isnot = s.find('not')
```

Comment:
Pythonic variables are snake-case. so, `is_not` would be appropriate here, and `is_bad` would be appropriate in the next line.